.gitignore

```diff
@@ -1,7 +1,2 @@
 *~
-.settings
```

Comment:
Hmm - possibly, but I think .gitignore is more of a de-facto way of managing this to prevent accidental commits with user settings etc.